app/actions/views/channel.js

```diff
@@ -255,15 +256,20 @@ export function handleSelectChannel(channelId) {
     return async (dispatch, getState) => {
         const {currentTeamId} = getState().entities.teams;
 
+        loadPostsIfNecessaryWithRetry(channelId)(dispatch, getState);
```

Comment:
I guess we're doing this here instead of in the channel screen somewhere? I think that makes a lot of sense